packages/react-components/src/helpers/format-date.js

```diff
@@ -0,0 +1,55 @@
+export const convertToDate = date => {
+  if (!date || !(date instanceof Date || typeof date === 'number')) return;
+  return new Date(date);
+};
+
+export const isDateAfter = (date1, date2) => {
+  const dateObj1 = convertToDate(date1);
+  const dateObj2 = convertToDate(date2);
+  if (!dateObj1 || !dateObj2) return;
+
+  return dateObj1.getTime() > dateObj2.getTime();
+};
+
+export const isDateBefore = (date1, date2) => {
+  const dateObj1 = convertToDate(date1);
+  const dateObj2 = convertToDate(date2);
+  if (!dateObj1 || !dateObj2) return;
+
+  return dateObj1.getTime() < dateObj2.getTime();
+};
+
+export const isDateToday = (date1, date2) => {
```

Comment:
This function isn't checking a date against `Date.now()`, so I think it would make more sense if the name more closely described the actual comparison that's happening.